packages/modules/imodel-browser/src/containers/ITwinGrid/useITwinData.ts

```diff
@@ -103,7 +103,8 @@ export const useITwinData = ({
     const endpoint = ["favorites", "recents"].includes(requestType)
       ? requestType
       : "";
-    const subClass = `?subClass=${iTwinSubClass}`;
+    const resolvedITwinSubClass = iTwinSubClass === "All" ? "" : iTwinSubClass;
+    const subClass = `?subClass=${resolvedITwinSubClass}`;
```

Comment:
thoughts on changing the default subclass from Projects to an empty string since subclass is optional?
while this is breaking in terms of prev behavior, its just widening the scope of subclasses pulled from projects to all

Reply:
Right, I played it safe with limited knowledge of how most of these subclasses are used. If "All" seems like a more sensible default, let's do it.

Reply:
@kckst8 @simnorm @simihartstein 
since you guys are the main consumers, do you have a preference?

Reply:
I'm not the best one to answer, but I don't know of any use-cases today where we would search for anything other than projects. Maybe we'll end up querying for portfolio iTwins once we get to catalogs, but Projects would be the norm.

If nothing else, I'd say leave it the same to avoid a breaking changes.